### PR TITLE
Fix stylistic issues, fix issue #32

### PR DESCRIPTION
**Pull request by vbmade2000**

### CODE REVIEW

**Comment by vbmade2000** [comment]
@jvehent Please take a look at PR.

File: cresponse/cresponse.go [hunk 1]
```diff
@@ -20,11 +20,11 @@ func (err CError) Error() string {
 	return fmt.Sprintf("System error number %d: %s", err.number, err.description)
 }
 
-// GetGetResponsesErrors returns the Go representation of the errors present in a C.reponse_t.
+// GetResponsesErrors returns the Go representation of the errors present in a C.response_t.
 //
 // NOTE: cgo types are private to each module, so exporting a function that expects a *C.response_t doesn't make sense,
 // so we export a function with an unsafe.Pointer and we cast it internally.
-func GetResponsesErrors(responsePointer unsafe.Pointer) (harderror error, softerrors []error) {
+func GetResponsesErrors(responsePointer unsafe.Pointer) (softerrors []error, harderror error) {
```
Comment (jvehent) [inline cresponse/cresponse.go]: I see you changed the order of the returned arguments in a number of places. Any reason why?

**Comment by jvehent** [comment]
Thanks for the patch!
Could you rebase against master? I added a travisci file that will run unit tests for us.

File: cresponse/cresponse.go (hunk 1 above)
Comment (vbmade2000) [inline cresponse/cresponse.go]: It is to resolve below golint warning. According to golint conventions, erro should be the last argument always.

> warning: error should be the last type when returning multiple items (golint)

File: memaccess/memaccess.go [hunk 2]
```diff
@@ -19,35 +19,35 @@ func (m MemoryRegion) String() string {
 	return fmt.Sprintf("MemoryRegion[%x-%x)", m.Address, m.Address+uintptr(m.Size))
 }
 
-// A centinel value indicating that there is no more regions available.
+// NoRegionAvailable is a centinel value indicating that there is no more regions available.
 var NoRegionAvailable MemoryRegion
 
 // NextReadableMemoryRegion returns a memory region containing address, or the next readable region after address in
```
Comment (vbmade2000) [inline memaccess/memaccess.go]: It is resolve following golint warning.

> warning: comment on exported var NoRegionAvailable should be of the form "NoRegionAvailable ..." (golint)

Any comment for exported variable or function should start with name of identifier.

**Comment by vbmade2000** [comment]
@jvehent I think I can't rebase as I don't have write access to this repo.

**Comment by jvehent** [comment]
`git pull` the master branch, then go back to your own branch and `git rebase origin/master`

**Comment by vbmade2000** [comment]
@jvehent Tried that but getting following issue.

> [vbmade2000@localhost masche]$ git checkout master 
> Switched to branch 'master'
> Your branch is up-to-date with 'origin/master'.
> [vbmade2000@localhost masche]$ git pull
> Already up-to-date.
> [vbmade2000@localhost masche]$ git checkout fix-style-issue 
> Switched to branch 'fix-style-issue'
> [vbmade2000@localhost masche]$ git rebase master 
> Current branch fix-style-issue is up to date.
> [vbmade2000@localhost masche]$ git commit --amend 
> [fix-style-issue bdf6fae] Fix stylistic issues, fix issue #32
>  Date: Tue Mar 21 12:48:25 2017 -0500
>  19 files changed, 114 insertions(+), 106 deletions(-)
> [vbmade2000@localhost masche]$ git push origin fix-style-issue  
> To https://github.com/vbmade2000/masche.git
>  ! [rejected]        fix-style-issue -> fix-style-issue (non-fast-forward)
> error: failed to push some refs to 'https://github.com/vbmade2000/masche.git'
> hint: Updates were rejected because the tip of your current branch is behind
> hint: its remote counterpart. Integrate the remote changes (e.g.
> hint: 'git pull ...') before pushing again.
> hint: See the 'Note about fast-forwards' in 'git push --help' for details.
> [vbmade2000@localhost masche]$ 
> 

There may be minor issue but I have no idea how should I overcome that.

**Pull request closed without merging**